editor/components/index.js

```diff
@@ -5,6 +5,7 @@ export { default as AlignmentToolbar } from './alignment-toolbar';
 export { default as BlockAlignmentToolbar } from './block-alignment-toolbar';
 export { default as BlockControls } from './block-controls';
 export { default as BlockEdit } from './block-edit';
+export { withBlockEditContext } from './block-edit/context';
```

Comment:
I'd prefer if we don't expose this one to avoid confusion with Block `edit` props for block authors. Hopefully we can get rid of `createInnerBlockList`

Reply:
Well, it would allow me to refactor `Gallery` and `Image` to remove focus when an image is clicked so I'm not that against exposing it :)

However, I think we agreed that we should avoid exposing this context if possible. If there is a better way to do it, then let's keep this context internal as long as necessary.

Reply:
If we merge #7199, we shouldn't need to export it. It's used exclusively by the shared block in its rendering of `<BlockEdit />` which needs `renderBlockMenu`. If we can make it work with just `uid` (already passed as a prop to block's `edit`), then it can be avoided.

Reply:
Let’s merge #7199 in that case 🎉